RELEASE/scripts/autoscend/auto_providers.ash

```diff
@@ -575,8 +577,8 @@ int [element] provideResistances(int [element] amt, location loc, boolean doEqui
 			cli_execute("speculate quiet; " + unequipsString);
 			foreach ele in amt
 			{
-				//add amount below goal that would be lost without current gear
-				amt[ele] += max(0,(min(amt[ele],numeric_modifier(ele + " resistance")) - simValue(ele + " resistance")));
+				//record the amount that would be lost to modify the result with
+				gearLoss[ele] = min(0,simValue(ele + " resistance") - numeric_modifier(ele + " resistance"));

```

Comment:
Should min be changed to max here? I would expect this value to be negative if gear provided this ele res